i18n/locales/en.json

```
@@ -249,6 +249,47 @@
     "source": "Data sourced from public breach notifications and security reports. Numbers represent approximate affected users/records.",
     "title": "ANZ Cyber Threats 2024"
   },
+  "v2": {
+    "app": {
```

Biome (1.9.4)

[error] The key v2 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)